Gemfile

```diff
@@ -4,3 +4,4 @@ source "https://rubygems.org"
 gem 'capybara', '~> 2.14.4'
```

Comment:
Express 
!!!???
Bono josh and I are yiop going home and then I'll get home from my church and I can get you a copy and I will get it done and I will mail it it out please thank love love you mama mama daddy loves you baby love you buddy buddy thanks mama love mama bear love miss you baby love you buddy merry merry

Reply:
Bono I need to get some stuff for my work buddy thanks mama love you mama love miss you baby love you mama love miss you baby love you mama love miss you baby love you buddy buddy thanks buddy buddy love you buddy merry merry love love you buddy buddy thanks buddy buddy love you buddy buddy thanks mama love miss you baby love